hw/ip/nmi_gen/doc/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "NMI Generator Technical Specification"
+---
+
+# Overview
+
+This document specifies the functionality of the non-maskable interrupt generator (`nmi_gen`) peripheral.
+This module conforms to the [OpenTitan guideline for peripheral device functionality.]({{< relref "doc/rm/comportability_specification" >}}).
+See that document for integration overview within the broader OpenTitan top level system.
+The module provides a mechanism to test the alert handler escalation signals (see [alert handler spec]({{< ref "hw/ip/alert_handler/doc" >}})), and will be modified or even replaced in the near future.
+
+
+## Features
+
+- Contains four escalation receivers to receive escalation signals from the alert handler
+
```

Comment:
probably don't need the empty line here.

Reply:
fixed